spec.html

```diff
@@ -1188,6 +1188,31 @@ <h1>
           <p>This algorithm always returns -1 if _fromIndex_ + the length of _searchValue_ > the length of _string_.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-stringlastindexof" type="abstract operation">
+        <h1>
+          StringLastIndexOf (
+            _string_: a String,
+            _searchValue_: a String,
+            _fromIndex_: a non-negative integer,
+          ): an integer
+        </h1>
+        <dl class="header">
+        </dl>
+        <emu-alg>
+          1. Let _len_ be the length of _string_.
+          1. Let _searchLen_ be the length of _searchValue_.
+          1. Assert: _fromIndex_ + _searchLen_ ≤ _len_.
+          1. If _searchValue_ is the empty String, return _fromIndex_.
```

Comment:
Why this? It's not in the original, it's not in StringIndexOf, and it's not necessary.

Reply:
Since this allows us to introduce a note saying the same thing, I'd prefer to get rid of this step, despite it being there in the original.

Reply:
Done.